obci/interfaces/bci/p300_MD/helper_functions.py

```diff
@@ -0,0 +1,503 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# based on obci.analysis.p300.analysis_offline
+# Marian Dovgialo
+from scipy import *
```

Comment:
no need to bother original author, just use imports from original file or fix your code :)